reddit_scraper.py

```
@@ -0,0 +1,111 @@
+    ]
+    creds = Credentials.from_service_account_file(
+        "secrets/service_account.json", scopes=scope
+    )
+    gc = gspread.authorize(creds)
+    sheet = gc.open(SPREADSHEET_NAME).sheet1
+    sheet = client.open(SPREADSHEET_NAME).sheet1
+    return sheet
+
+
+def run_scraper(search_term):
+    urls = google_search(f"site:reddit.com {search_term} pain point")
```

Ruff (0.11.9)

Local variable `urls` is assigned to but never used

Remove assignment to unused variable `urls`

(F841)